release.Dockerfile

```diff
@@ -1,33 +1,42 @@
+FROM python:3.9-slim AS builder
+
+# Copying argilla distribution files
+COPY dist/*.whl /packages/
+RUN python -m venv /my-virtual-env
+ENV PATH="/my-virtual-env/bin:$PATH"
+RUN apt-get update && \
+    apt-get install -y python-dev-is-python3 libpq-dev gcc && \
```

Comment:
I couldn't get `python-dev` to install so changed to `python-dev-is-python3`

Reply:
Yes, the latest python base image upgraded the OS, and the `python-dev` is not working anymore. Thanks